prisma/schema/teamStack.prisma

```diff
@@ -2,8 +2,10 @@
 
 model TechStackCategory {
   id          Int    @id @default(autoincrement())
-  name        String @unique
+  name        String 
   description String
+  voyageTeam         VoyageTeam?         @relation(fields: [voyageTeamId], references: [id], onUpdate: Cascade, onDelete: Cascade)
+  voyageTeamId       Int? 
```

Comment:
will voyageTeamId ever be null?

Reply:
No - it won't ever be null.  I made this change because it was failing the build check when I pushed to github, and this was the easiest solution.  I think it said we can also make the DB reset when building

Reply:
yeah at this stage it won't matter, since we are not actually using the prod database. We could re seed it again. But I think in future when we have cases like this (create a new non nullable column with existing null values), which is vey likely, we need to do something like

1. create a nullable newColumn, migrate, fill it with values manually or with a script
2. make newColumn non-nullable, migrate

or 
1. create a non-nullable newColumn with a default value, migrate
2. remove default, migrate 

This will preserve the original data. We will need to do this in 2 stages, so it would require 2 PRs to pass the build checks.

If we want to do this, we can do step 1 in this PR, then after it's merged, we can do another for step 2

Reply:
Or we could just do a PR now which would include the first step - so we can setup uniqueness in this PR (with step 2)

Reply:
So - since the column is already nullable , we can fill null values manually or set a default value and migrate?

Reply:
hmm you can try but I thought the migration was getting an error, thats why we have to do a 2 step migration

Reply:
Ok - I added a voyageTeamIdNew column to the tech stack category schema - no default, and a relation for it